roadmap.md

```diff
@@ -0,0 +1,35 @@
+Roadmap Zombienet v2
+
+## Infra
```

Comment:
Prometheus server deployment -> matching against prometheus queries. 
- we remove code and inefficiency from scraping metrics ourselves
- scales better
- we get a standardized way of querying metrics and ability to do aggregations as we see fit
- (nice to have) create some test reports using these, maybe alarms for long duration failure conditions
- local debugability increases

Open question:
- do we use prometheus for native/local runs, or just k8s ? (if this differers across providers, prometheus queries won't work)

Reply:
I think local prometheus should be `opt-in` and not a default.